metadata.yaml

```diff
@@ -37,6 +41,10 @@ provides:
     limit: 1
 
 requires:
+  async-replica:
+    interface: async_replication
+    limit: 1
+    optional: true
```

Comment:
as followup commit: let's move to agreed `async` here.

Reply:
Sure. I created the following ticket for that: https://warthogs.atlassian.net/browse/DPE-4257